### DISCUSSION TITLE
Quick question from new user.

### DISCUSSION THREAD

**Discussion by aggrophobik** [memstechtips/UnattendedWinstall#141] [Q&A]
First off, thank you so much for your time in creating/maintaining this, it's awesome! I'm fairly new to using unattended install method and came across this instead of making my own. My main question is would I want to run Chris Titus's "tweaks" section and set up updates with his tool, or is that mostly redundant?  I am using Standard xml and was reading that it has quite a few tweaks already baked in. I didn't want to mess anything up running his tool/tweaks on top of this. I hope I was able to articulate that to make sense. Thanks in advance for any help or advice!

**Comment by memstechtips** [comment]
Hey there and sorry for the late reply,

Since the creation of this issue I have released a new version of the XML file and a major update to [Winhance](https://winhance.net/), I recommend that you watch [this video](https://youtu.be/lrq3ph3xi50) which provides a full demo of the current version along with how to use the new version of WIMUtil and how to install the drivers etc. You now also have the ability to create an autounattend.xml file based on your selections in Winhance, so you have more control over which apps are removed and which settings are applied.

I have also created [this video](https://youtu.be/sCd1TI6AbOI) about v3.0.0 of UnattendedWinstall, I suggest that you watch it too if you're interested.

Btw, you can still use Chris Titus WinUtil, it won't clash with anything.

I'm closing this issue now, but if you need more help with the same issue, feel free to reopen it.
